.github/workflows/ci.yaml

```diff
@@ -1,8 +1,6 @@
 name: ci
 on:
-  pull_request:
-    branches:
-      - main
+  pull_request: {}
```

Comment:
ci workflow wasn't running because we only run against `main` and I'm working again`release-candidate/2024-10` here.